multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/jobs/OrphanedFilesCleaner.java

```diff
@@ -0,0 +1,63 @@
+package org.cloudfoundry.multiapps.controller.process.jobs;
+
+import java.text.MessageFormat;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.cloudfoundry.multiapps.common.SLException;
+import org.cloudfoundry.multiapps.controller.core.util.ApplicationConfiguration;
+import org.cloudfoundry.multiapps.controller.persistence.model.FileEntry;
+import org.cloudfoundry.multiapps.controller.persistence.services.AsyncUploadJobService;
+import org.cloudfoundry.multiapps.controller.persistence.services.FileService;
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+
+@Named
+public class OrphanedFilesCleaner {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrphanedFilesCleaner.class);
+    private static final int SELECTED_INSTANCE_FOR_CLEAN_UP = 1;
```

Comment:
we already have a cleaner FinishedFlowableHistoricProcessesCleaner, that works on instance 1. We must always run on at least 3 instances because of multi AZ, so I do not think it is an issue. Restarting the instance, retriggers the job but this is not an issue, only the files in the previous hour will be removed without started operation